html2print/html2print.py

```diff
@@ -305,34 +329,44 @@ class Done(Exception):
 
 
 def create_webdriver(
+    browser_info: browsers.Browser,
     chromedriver: Optional[str],
     path_to_cache_dir: str,
     page_load_timeout: int,
     debug: bool = False,
 ) -> webdriver.Chrome:
     print("html2print: creating ChromeDriver service.", flush=True)  # noqa: T201
+
     if chromedriver is None:
-        path_to_chrome = ChromeDriverManager().get_chrome_driver(
-            path_to_cache_dir
+        path_to_chrome_driver = ChromeDriverManager().get_chrome_driver(
+            chrome_version=browser_info["version"],
+            path_to_cache_dir=path_to_cache_dir,
         )
     else:
-        path_to_chrome = chromedriver
-    print(f"html2print: ChromeDriver available at path: {path_to_chrome}")  # noqa: T201
+        path_to_chrome_driver = chromedriver
+    print(  # noqa: T201
+        f"html2print: ChromeDriver available at path: {path_to_chrome_driver}"
+    )
 
     if debug:
         service = Service(
-            path_to_chrome, log_output=PATH_TO_CHROME_DRIVER_DEBUG_LOG
+            path_to_chrome_driver, log_output=PATH_TO_CHROME_DRIVER_DEBUG_LOG
         )
     else:
-        service = Service(path_to_chrome)
+        service = Service(path_to_chrome_driver)
 
     webdriver_options = Options()
+    # Workaround for Windows: chrome is not typically found in the PATH, so need to supply the exact binary location manually
+    if platform.system() == "Windows":
+        webdriver_options.binary_location = browser_info["path"]
     webdriver_options.add_argument("start-maximized")
     webdriver_options.add_argument("disable-infobars")
     # Doesn't seem to be needed.
     # webdriver_options.add_argument('--disable-gpu')  # noqa: ERA001
     webdriver_options.add_argument("--disable-extensions")
-    webdriver_options.add_argument("--headless=chrome")
+    # Use --headless=new, as it seems to be more stable on Windows (available since Chrome 109).
+    # see https://www.selenium.dev/blog/2023/headless-is-going-away/
+    webdriver_options.add_argument("--headless=new")
     # FIXME: This is not nice but otherwise it does not work in Ubuntu 24-based Docker image.
     # https://github.com/SeleniumHQ/selenium/issues/15327#issuecomment-2689287561
     webdriver_options.add_argument("--no-sandbox")
```

Comment:
I have been trying to re-enable the sandbox inside the Docker container ,but i turns out that Chrome's sandbox feature relies on some syscalls that are normally disabled in the container. 
https://stackoverflow.com/questions/68855734/how-to-setup-chrome-sandbox-on-docker-container.

Enabling these would improve the isolation between invidual tabs within the same chrome instance, while at the same time reducing the isolation between chrome within the container, and its host. As in the use casel of html2print, there should not be any other open tabs, and the container is read-only, I would say that disabling the sandbox and keeping maxmum container isolation is the better option.

If you agree, I would update the comment to something like:
```suggestion
    # Docker disables some syscalls that are required for Chrome's sandbox to work.
    # https://stackoverflow.com/questions/68855734/how-to-setup-chrome-sandbox-on-docker-container.
    # We prefer isolation of the container over isolation of tabs within Chrome, and thus disable the sandbox 
    webdriver_options.add_argument("--no-sandbox")
```